test/Interop/Cxx/templates/function-template.swift

```diff
@@ -28,15 +28,6 @@ FunctionTemplateTestSuite.test("lvalueReference<T> where T == Int") {
   expectEqual(value, 42)
 }
 
-// TODO: currently "Any" is imported as an Objective-C "id".
```

Comment:
This might look like a regression, but this should never have really been tested. It just happens to work. It was never designed, implemented, or meant to be supported.